### ISSUE TITLE
Plugin 5.0.2+ isLocationAuthorized throws error on Android 6.0

### ISSUE THREAD

**Issue by WeaponX86** [dpa99c/cordova-diagnostic-plugin#423]
# Bug report

**CHECKLIST**
- [x] I have read the [issue reporting guidelines](https://github.com/dpa99c/cordova-diagnostic-plugin#reporting-issues)

- [x] I confirm this is a suspected bug or issue that will affect other users

- [x] I have reproduced the issue using the [example project](https://github.com/dpa99c/cordova-diagnostic-plugin-example) or provided the necessary information to reproduce the issue.

- [x] I have read [the documentation](https://github.com/dpa99c/cordova-diagnostic-plugin) thoroughly and it does not help solve my issue.

- [x] I have checked that no similar issues (open or closed) already exist.

**Current behavior:**
cordova.plugins.diagnostic.isLocationAuthorized error callback throws:

> Error: string arg == null

**Expected behavior:**
cordova.plugins.diagnostic.isLocationAuthorized should return a boolean. Works fine in v5.0.1.

**Steps to reproduce:**
Built example app from https://github.com/dpa99c/cordova-diagnostic-plugin-example. Added the following settings:

`<preference name="cordova-ios-version" value="6.1.1" />
    <preference name="cordova-android-version" value="9.0.0" />
  <preference name="android-minSdkVersion"      value="23" />
  <preference name="android-targetSdkVersion"   value="29" />
`

**Screenshots**
[https://pasteboard.co/JzBNzFR.png](https://pasteboard.co/JzBNzFR.png)

**Environment information**

- Cordova CLI version 
	- 10.0.0
- Cordova platform version
	- cordova-android 9.0.0
- Plugins & versions installed in project (including this plugin)

> cordova-plugin-buildinfo 4.0.0 "BuildInfo"
> cordova-plugin-device 2.0.3 "Device"
> cordova-plugin-file 6.0.2 "File"
> cordova-plugin-geolocation 4.0.2 "Geolocation"
> cordova-plugin-inappbrowser 4.0.0 "InAppBrowser"
> cordova-plugin-network-information 2.0.2 "Network Information"
> cordova-plugin-screen-orientation 3.0.2 "Screen Orientation"
> cordova-plugin-statusbar 2.4.3 "StatusBar"
> cordova-plugin-whitelist 1.3.4 "Whitelist"
> cordova.plugins.diagnostic 5.0.2 "Diagnostic"
> es6-promise-plugin 4.2.2 "Promise"

- Dev machine OS and version, e.g.
    - OSX
        - macOS 10.15.7
        - xcode 12.1
        
_Runtime issue_
- Device details
    - Samsung Galaxy S5
- OS details
    - Android 6.0.1

**Related code:**
```
cordova.plugins.diagnostic.isLocationAuthorized(function(b){
	console.log('b: ' + b);
},function(e){
	console.error(e);
});
```

**Console output**
<details>
<summary>console output</summary>

```
string arg == null

```

</details><br/><br/>

**Other information:**

**Comment by akimakitalo** [comment]
getLocationAuthorizationStatus() also returns "string arg == null".

Probably everything related to location authorization does this?

**Comment by infacto** [comment]
We call `isLocationAuthorized` and `isLocationEnabled`. Same error. 
@dpa99c Any guesses?

**Comment by fcamblor** [comment]
Same here :
- Android 6.0.1
- `cordova-diagnostic-plugin`@`6.0.2`
- Chrome Mobile WebView 44.0.2403


Logcat stacktrace :
```
12-03 15:47:20.449 10489-10561/com.chep.carrier.roundtrips V/Diagnostic: Get authorisation status for android.permission.ACCESS_COARSE_LOCATION
12-03 15:47:20.450 10489-10561/com.chep.carrier.roundtrips V/Diagnostic: Get authorisation status for android.permission.ACCESS_FINE_LOCATION
12-03 15:47:20.450 10489-10561/com.chep.carrier.roundtrips V/Diagnostic: Get authorisation status for android.permission.ACCESS_BACKGROUND_LOCATION
12-03 15:47:20.451 10489-10561/com.chep.carrier.roundtrips E/PluginManager: Uncaught exception from plugin
    java.lang.NullPointerException: string arg == null
        at java.lang.String.concat(Native Method)
        at cordova.plugins.Diagnostic.execute(Diagnostic.java:256)
        at org.apache.cordova.CordovaPlugin.execute(CordovaPlugin.java:98)
        at org.apache.cordova.PluginManager.exec(PluginManager.java:132)
        at org.apache.cordova.CordovaBridge.jsExec(CordovaBridge.java:59)
        at org.apache.cordova.engine.SystemExposedJsApi.exec(SystemExposedJsApi.java:41)
        at org.chromium.base.SystemMessageHandler.nativeDoRunLoopOnce(Native Method)
        at org.chromium.base.SystemMessageHandler.handleMessage(SystemMessageHandler.java:53)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.os.HandlerThread.run(HandlerThread.java:61)
```

**Comment by fcamblor** [comment]
Some additionnal informations regarding error above, I updated a bit the exception handling to have more informations on the underlying exception and here we go :

<img width="358" alt="Android_Emulator_-_Pixel_2_API_23_5554" src="https://user-images.githubusercontent.com/603815/101172678-c58c1c00-3641-11eb-9752-d3a51aed42e2.png">

**Comment by fcamblor** [comment]
@dpa99c is there anything I can do to help ?

I tried to dig into changes made between `5.0.1` and `5.0.2` which introduced the regression, but I'm lacking a lot of knowledge and I fear doing more harm than good.

**Comment by ath0mas** [comment]
Hello, can I ask if the corresponding release is planned?

**Comment by dpa99c** [comment]
Released in v6.0.3

**Comment by ath0mas** [comment]
👍 

just a typo for the Changelog: `Android API 29` means **10.0**, not _11.0_
